handlers/handlers.go

```diff
@@ -74,8 +74,8 @@ func setAuthTokenIfRequired(req *http.Request) ([]dataset.Config, []filter.Confi
 	var filterConfig []filter.Config
 	if len(req.Header.Get("X-Florence-Token")) > 0 {
```

Comment:
Would it be easier to read if `florenceToken := req.Header.Get("X-Florence-Token")` and then use florenceToken on line 75 and 78?